### PR TITLE
Changes based on gen-art review by Paul K.

### PR DESCRIPTION
**Pull request by cdh4u**

### CODE REVIEW

File: draft-ietf-mmusic-dtls-sdp.xml [hunk 1]
```diff
@@ -280,7 +280,7 @@
            </t>
             <t>
                 When an offerer or answerer indicates that it wants to establish a new DTLS association, it needs to make sure that
-                media packets in the existing DTLS association and new DTLS association can be de-multiplexed. In case
```
Comment (rshpount) [inline draft-ietf-mmusic-dtls-sdp.xml:283]: I think we  agreed to say "media packets in any previously established DTLS association" not "any existing".

Comment (cdh4u) [reply draft-ietf-mmusic-dtls-sdp.xml:283]: In the e-mail thread I suggested "any existing". But, I agree it should say "any previously established".

Comment (rshpount) [reply draft-ietf-mmusic-dtls-sdp.xml:283]: I was referring to https://www.ietf.org/mail-archive/web/mmusic/current/msg17781.html

Comment (cdh4u) [reply draft-ietf-mmusic-dtls-sdp.xml:283]: You are right - I'm sorry I missed that when implementing the changes. Should be fixed now, in the latest commit.

**Pull request merged**